bootstrap/bootstrap.go

```diff
@@ -1120,7 +1122,7 @@ func (b *Bootstrap) defaultCheckoutPhase() error {
 		b.shell.Commentf("Fetch and checkout pull request head from GitHub")
 		refspec := fmt.Sprintf("refs/pull/%s/head", b.PullRequest)
 
-		if err := gitFetch(b.shell, "-v", "origin", refspec); err != nil {
```

Comment:
This is the only case we don't include `--prune` @sj26, do you happen to recall why?

Reply:
I think we just never added it 🤷‍♂️